libs/features/advert-page/src/lib/features-advert-page.tsx

```diff
@@ -71,6 +72,7 @@ let lessons = [
 let lessonsDescription =
   'Are you ready to find out what all the hype is about with ReactJS? These ReactJS for beginners tutorials will bring you completely up to speed on the hottest JavaScript framework used on the web today.';
 
+const accordionTitle = 'Lessons';
```

Comment:
If title of accordion is hardcoded, why do we need to assign it to variable? Just provide string directly to component.

Reply:
It allows easier to reuse the same shared component. For example, you use it as "Lessons", but I use it as "Related lessons".